### ISSUE TITLE
ClassCastException while trying to iterate over performance metrics

### ISSUE THREAD

**Issue by varun85gupta** [kklisura/chrome-devtools-java-client#21]
While trying to iterate the list from _getMetrics_ in Performance interface, I am encountering this error :

java.util.LinkedHashMap cannot be cast to com.github.kklisura.cdt.protocol.types.performance.Metric"

Sample code: https://gist.github.com/varun85gupta/7ceab06ad7db39a45f8c49454b0dcf21

**Comment by varun85gupta** [comment]
Although we can still achieve it using this, it would be better if we can iterate directly.
```
for(Object metric: metrics)
{
       LinkedHashMap<String, Double> metric1 = (LinkedHashMap<String, Double>) metric;
       System.out.println(metric1.get("name")+":"+ metric1.get("value"));
}
```

**Comment by kklisura** [comment]
Hi, thanks for using this library and for opening this issue. The fix for this should be available in `1.3.6` that has just been released. You can use `getMetrics` (and similar methods) without that casting.

**Comment by varun85gupta** [comment]
Thanks for the fix. I'll validate it.
